### PR TITLE
FEATURE: Add validation

### PR DESCRIPTION
**Pull request by p0pl3**
dont send it for rework pls

### CODE REVIEW

File: src/components/UserAuthForm.vue [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
-    <h1><slot name="title"></slot></h1>
+    <h1>
+      <slot name="title"></slot>
+    </h1>
     <form @submit.prevent>
-      <my-input class="input__login" placeholder="Username" v-model="userLog.username"/>
-      <my-input class="input__login" placeholder="Password" v-model="userLog.password" type="password"/>
-      <my-button class="btn__login" @click="submitForm"><slot name="btn_form"></slot></my-button>
+      <my-input class="input__login" placeholder="Username" v-model="v$.userLog.username.$model"/>
+      <p class="helper_text" v-if="!v$.userLog.username.$error">Username</p>
```
Comment (Blastz13) [inline src/components/UserAuthForm.vue]: Please fix this moment, this inscription should not be

File: src/components/UserAuthForm.vue [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
+      <p class="error_message" v-for="error in v$.userLog.username.$errors">{{ error.$message }}</p>
+      <my-input class="input__login" placeholder="Password" v-model="v$.userLog.password.$model" type="password"/>
+      <p class="helper_text" v-if="!v$.userLog.password.$error">Password</p>
```
Comment (Blastz13) [inline src/components/UserAuthForm.vue]: And this inscription should not be

File: src/components/UserAuthForm.vue [hunk 3]
```diff
@@ -27,8 +40,20 @@ export default {
   },
   methods: {
     submitForm() {
+      if (this.v$.$invalid) {
```
Comment (Blastz13) [inline src/components/UserAuthForm.vue]: Fix the logic of form validation and remove this trash

**Review by Blastz13 — COMMENTED** [review]
Good job! I like it, but please fix the issues with window resizing when form validation errors occur.

**Comment by p0pl3** [comment]
I fix all that you want.

File: src/components/UserAuthForm.vue [hunk 4]
```diff
@@ -13,22 +21,43 @@
...
-      }
+      },
+      passwordValidationError: false,
+      usernameValidationError: false
     }
   },
   methods: {
     submitForm() {
+      if (this.v$.$invalid) {
+        this.usernameValidationError = this.v$.userLog.username.$error;
```
Comment (Blastz13) [inline src/components/UserAuthForm.vue]: Fix the form validation logic on submit, these fields are not needed here

**Review by Blastz13 — COMMENTED** [review]
Everything is great, just fix some minor flaws

**Comment by p0pl3** [comment]
Delete useless fields;
Code review.

**Comment by Blastz13** [comment]
Well done, thanks for the contribution!

**Pull request merged**